activerecord/test/models/user.rb

```diff
@@ -10,6 +10,8 @@ class User < ActiveRecord::Base
     class_name: "Job",
     join_table: "jobs_pool"
 
+  has_one :room
+  has_one :owned_room, class_name: "Room", foreign_key: "owner_id"
```

Comment:
Let's remove the unused `owned_room` association.

Reply:
ah it is used in the line, nvm.

https://github.com/rails/rails/blob/1f969fc944bdeffc1518edd6e22b31680816ffea/activerecord/test/cases/associations/inverse_associations_test.rb#L98